go/teams/chatmessaging.go

```diff
@@ -49,16 +49,10 @@ func SendTeamChatWelcomeMessage(ctx context.Context, g *libkb.GlobalContext, tea
 	}
 	username := g.Env.GetUsername()
 	subBody := chat1.NewMessageSystemWithAddedtoteam(chat1.MessageSystemAddedToTeam{
-		Adder:          username.String(),
-		Addee:          user,
-		Role:           role,
-		Team:           team,
-		Owners:         ownerNames,
```

Comment:
dont' need to get `TeamDetails` either, can delete a bunch of the above code. Why not remove this from the message struct?